load-testing/initial-load-testing.md

```diff
@@ -0,0 +1,93 @@
+# Initial load testing
+
+## Files used
+
+A 3Gb, 40000 second wav file, created using ffmpeg.
+
+2 2.8Gb bmp files created using ImageMagick's convert tool
+
+2000 PDFs created using ImageMagick's convert tool
+
+10000 PDFs created using ImageMagick's convert tool
```

Comment:
Probably not a problem given the types of files that the first users will be transferring, but did you ever test it with purely random data? i.e. files that DROID wouldn't be able to identify?

Reply:
I did yeah, I had a 5Gb file generated from random bytes. It's in the table but not in the list at the beginning. I've added it in here.